libs/sdk-ui-filters/src/DateFilter/DateFilterCore.tsx

```diff
@@ -138,20 +144,38 @@ export const DateFilterCore: React.FC<IDateFilterCoreProps> = ({
                                 ".s-do-not-close-dropdown-on-click",
                                 ".rdp-day", // absolute range picker calendar items
                             ]}
-                            renderBody={({ closeDropdown }) => (
-                                // Dropdown component uses React.Children.map and adds special props to this component
-                                // https://stackoverflow.com/questions/32370994/how-to-pass-props-to-this-props-children
-                                <DateFilterBody
-                                    {...dropdownBodyProps}
-                                    filterOptions={filteredFilterOptions}
-                                    isMobile={isMobile}
-                                    closeDropdown={closeDropdown}
-                                    dateFilterButton={dateFilterButton()}
-                                    dateFormat={verifiedDateFormat}
-                                    isTimeForAbsoluteRangeEnabled={isTimeForAbsoluteRangeEnabled}
-                                    weekStart={weekStart}
-                                />
-                            )}
+                            renderBody={({ closeDropdown }) => {
+                                return isConfigurationOpen ? (
+                                    <FilterConfigurationComponent
+                                        onSaveButtonClick={() => {
+                                            setIsConfigurationOpen(false);
+                                        }}
+                                        onCancelButtonClick={() => {
+                                            setIsConfigurationOpen(false);
+                                            onCancelClick();
```

Comment:
The inline function makes the component be rerenderred, please create a separate function for this prop!